ME/Extension/UIColor+Extension.swift

```diff
@@ -8,11 +8,11 @@
 import UIKit
 
 extension UIColor {
-    
     @nonobjc class var background: UIColor { return UIColor(named: "background")! }
+    @nonobjc class var darkGray: UIColor { return UIColor(named: "darkGray")! }
     @nonobjc class var gray: UIColor { return UIColor(named: "gray")! }
+    @nonobjc class var lightGray: UIColor { return UIColor(named: "lightGray")! }
     @nonobjc class var yellow: UIColor { return UIColor(named: "yellow")! }
     @nonobjc class var lightOrange: UIColor { return UIColor(named: "lightOrange")! }
```

Comment:
```suggestion
    static let background = UIColor(resource: .background)
    static let darkGray =  UIColor(resource: .darkGray)
    static let gray = UIColor(resource: .gray)
    static let lightGray = UIColor(resource: .lightGray)
    static let yellow = UIColor(resource: .yellow)
    static let lightOrange = UIColor(resource: .lightOrange)
```
`@noobjc`의 경우 Objective-C에서 접근이 불가능하다는 걸 명시적으로 나타내는 역할을 하는데, 현재 프로젝트는 100% Swift 기반으로 만들어져 있기 때문에 굳이 사용할 필요가 없지 않을까? 대신 `static let` 형식으로 선언해서 사용하는 게 좋을 거 같네~

Color 또한 Xcode 15 업데이트 이후 ColorResource 접근이 가능해졌으니 String 형식 대신 사용해보자!